File: app/src/main/java/org/jesperancinha/xml/adder/csv/AttributeAddBean.java
package org.jesperancinha.xml.adder.csv;


import com.opencsv.bean.CsvBindByName;
import com.opencsv.bean.CsvBindByNames;

/**
 * Created by joaofilipesabinoesperancinha on 18-02-16.
 */
public class AttributeAddBean {

    @CsvBindByName(capture = "name")
    private String name;

    @CsvBindByName(capture = "value")
    private String value;

    @CsvBindByName(capture = "xpath")
    private String xpath;

    public String getName() {
        return name;
    }

    public String getValue() {
        return value;
    }

    public String getXpath() {
        return xpath;
    }
}

File: jeorg-kotlin-apps/xml-adder/app/src/main/kotlin/org/jesperancinha/xml/adder/csv/AttributeAddBean.kt
package org.jesperancinha.xml.adder.csv

import com.opencsv.bean.CsvBindByName
import com.opencsv.bean.CsvBindByPosition

/**
 * Created by joaofilipesabinoesperancinha on 18-02-16.
 */
class AttributeAddBean {
    @CsvBindByName(capture = "name")
    @CsvBindByPosition(position = 0)
    var name: String? = null

    @CsvBindByName(capture = "value")
    @CsvBindByPosition(position = 1)
    var value: String? = null

    @CsvBindByName(capture = "xpath")
    @CsvBindByPosition(position = 2)
    var xpath: String? = null
}